### PR TITLE
Dalamud/ClientStructs API updates

### PR DESCRIPTION
**Pull request by electr0sheep**
Updates for DT

### CODE REVIEW

**Comment by electr0sheep** [comment]
Need to figure out what to do about ContextMenus and XIVCommon. The project relied on both of these, and both have not been updated. I updated XIVCommon myself, and everything seems to be working, but we should probably wait for both of these nuget packages to be updated.

**Comment by electr0sheep** [comment]
ContextMenus were fixed before Dalamud release, still no update on xivcommon, so I just copied over what the plugin needed

File: ItemVendorLocation/ItemVendorLocation.csproj [hunk 1]
```diff
@@ -15,9 +15,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="CheapLoc" Version="1.1.8" />
-		<PackageReference Include="Dalamud.DrunkenToad" Version="1.8.15" />
-		<PackageReference Include="DalamudPackager" Version="2.1.12" />
-		<PackageReference Include="XivCommon" Version="9.0.0" />
+		<PackageReference Include="Dalamud.DrunkenToad" Version="1.9.2" />
```
Comment (Nukoooo) [inline ItemVendorLocation/ItemVendorLocation.csproj:18]: I think this can be removed since I don't see anything from this package is used

**Comment by Nukoooo** [comment]
Looks good to me

**Pull request merged**